gnd/src/main/java/com/google/android/gnd/model/basemap/tile/TileSource.java

```diff
@@ -37,6 +42,8 @@
 
   public abstract State getState();
 
+  public abstract int getAreaCount();
```

Comment:
Nit: Calling it "reference count" might make the purpose of this field more obvious.

Reply:
I actually find `referenceCount` more ambiguous--references to what, exactly?

Maybe `associatedBasemapCount` or `basemapReferenceCount`?

Reply:
Changed to `basemapReferenceCount`